### PR TITLE
Fix/soc 5487

### PR DESCRIPTION
**Pull request by aboughzela**
update configuration.properties and exo-sample.properties to be able to disable activity types
and register a custom activity type.

### CODE REVIEW

File: extension/config/src/main/resources/conf/platform/exo-sample.properties [hunk 1]
```diff
@@ -692,6 +692,23 @@
...
+#
+
+# To disable any Activity Type add exo.activity-type.<activity-type-key>.enabled=false
+# by default all activity Type are activated.
+# The list of all activity type key {DEFAULT_ACTIVITY,SPACE_ACTIVITY,USER_ACTIVITIES_FOR_SPACE,exosocial:relationship,
+# LINK_ACTIVITY,sharecontents:spaces,USER_PROFILE_ACTIVITY,DOC_ACTIVITY,files:spaces,sharefiles:spaces,contents:spaces,
+# cs-calendar:spaces,TaskAdded,ks-forum:spaces,ks-answer:spaces,ks-poll:spaces,ks-wiki:spaces,USER_ACTIVITIES_FOR_RELATIONSHIP}
+#
+# exo.activity-type.DOC_ACTIVITY.enabled=true
+# exo.activity-type.cs-calendar:spaces.enabled=true
```
Comment (boubaker) [inline extension/config/src/main/resources/conf/platform/exo-sample.properties]: List all activity types in comment, for example:

+# exo.activity-type.DOC_ACTIVITY.enabled=true
 +# exo.activity-type.cs-calendar:spaces.enabled=true
 +# exo.activity-type.ks-poll:spaces.enabled=true
 +# exo.activity-type.USER_ACTIVITIES_FOR_RELATIONSHIP.enabled=true
...

File: extension/config/src/main/resources/conf/platform/configuration.properties [hunk 2]
```diff
@@ -663,6 +663,16 @@ wiki.editPage.livingTime=${exo.wiki.editPage.livingTime:1800000}
 
 ###########################
 #
+# social
+#
+# To disable any Activity Type add : exo.activity-type.<activity-type-key>.enabled=false
+# by default all activity Types are activated.
+# The list of all activity types defined by:
+
+exo.social.activity.types= ${exo.social.activity.types:DEFAULT_ACTIVITY,SPACE_ACTIVITY,USER_ACTIVITIES_FOR_SPACE,exosocial:relationship,LINK_ACTIVITY,sharecontents:spaces,USER_PROFILE_ACTIVITY,DOC_ACTIVITY,files:spaces,sharefiles:spaces,contents:spaces,cs-calendar:spaces,TaskAdded,ks-forum:spaces,ks-answer:spaces,ks-poll:spaces,ks-wiki:spaces,USER_ACTIVITIES_FOR_RELATIONSHIP}
```
Comment (boubaker) [inline extension/config/src/main/resources/conf/platform/configuration.properties]: Add 
,${exo.social.activity.custom.types}
at the end.

**Review by boubaker — CHANGES_REQUESTED** [review]

File: extension/config/src/main/resources/conf/platform/configuration.properties (hunk 2 above)
Comment (aboughzela) [reply extension/config/src/main/resources/conf/platform/configuration.properties]: exo.social.activity.custom.types added in exo-sample.properties

File: extension/config/src/main/resources/conf/platform/exo-sample.properties (hunk 1 above)
Comment (aboughzela) [reply extension/config/src/main/resources/conf/platform/exo-sample.properties]: Done

File: extension/config/src/main/resources/conf/platform/configuration.properties (hunk 2 above)
Comment (thomasdelhomenie) [inline extension/config/src/main/resources/conf/platform/configuration.properties]: What is the goal of this property and what is the impact for a developer who wants to add its own custom activity type ?

**Review by thomasdelhomenie — APPROVED** [review]

**Pull request closed without merging**